Samples/2.0/docker-aspnet-core/Makefile

```diff
@@ -0,0 +1,47 @@
+# a Makefile to test this sample on Kubernetes
+# you can test it locally (e.g. using Docker for Windows/Mac) or remotely, on your cloud provider of choice
+# do not forget to set Azure Storage connection string on Silo/Program.cs and API/Startup.cs
+
+# your Docker registry (we suppose you have logged in and have push access)
+REGISTRY ?= docker.io/dgkanatsios
+# the context of your local Kubernetes cluster
+LOCAL_K8S_CLUSTER=docker-for-desktop
+# the contect of your remote Kubernetes cluster
+REMOTE_K8S_CLUSTER=aksorleans
+# version/tag of the images that will be pushed to Docker Hub
+VERSION=0.0.14
+
+API_PROJECT_NAME=orleans-api
+SILO_PROJECT_NAME=orleans-silo
+
+# tag of the images that will be pushed for local development
+TAG?=$(shell git rev-list HEAD --max-count=1 --abbrev-commit)
+
+buildlocal: 
+		docker build -f ./Silo/Dockerfile -t $(SILO_PROJECT_NAME):$(TAG) .
+		docker build -f ./API/Dockerfile -t $(API_PROJECT_NAME):$(TAG) .
+		docker system prune -f
+deploylocal: uselocalcontext
+		kubectl config use-context $(LOCAL_K8S_CLUSTER)
+		sed -e 's/orleans-silo-image/$(SILO_PROJECT_NAME):$(TAG)/' -e 's/orleans-api-image/$(API_PROJECT_NAME):$(TAG)/' deploy.yaml | kubectl apply -f -
```

Comment:
Does `sed` will work on a Windows machine with just docker and kubectl installed?

Reply:
Nope, but `make` will not work either. Maybe add `use make if on Linux/WSL`?

Reply:
I only saw that now... I would avoid use make and sed for those examples... better use msbuild...